snakefile.py

```diff
@@ -376,6 +374,7 @@ def render_qc_report_params(wildcards, input, output = None, threads = None, res
 KRAKEN_DIR        = os.path.join(OUTPUT_DIR, 'kraken')
 COVERAGE_DIR      = os.path.join(OUTPUT_DIR, 'coverage')
 REPORT_DIR        = os.path.join(OUTPUT_DIR, 'report')
+FASTP_DIR         = os.path.join(OUTPUT_DIR, 'fastp')
 FASTQC_DIR        = os.path.join(REPORT_DIR, 'fastqc')
```

Comment:
Is FASTQC_DIR still used anywhere?  Or is it all FASTP_DIR now?

Reply:
Yes, I just wanted to make sure that the fastp output can be easily found. I felt it was confusing to put the output of one tool into the dir of another.